crates/swc_plugin_runner/Cargo.toml

```diff
@@ -58,8 +57,13 @@ serde        = { version = "1.0.126", features = ["derive"] }
 serde_json   = "1.0.64"
 tokio        = { version = "1", default-features = false, optional = true }
 tracing      = "0.1.37"
-wasmer       = { version = "4.0.0", default-features = false }
-wasmer-wasix = { version = "0.9.0", default-features = false }
+wasmer       = { version = "4.1.2", default-features = false, features = [
```

Comment:
I couldn't make it compile without these features

Reply:
This was intentional

Reply:
Please revert this and use cargo features defined in `swc_core`

Reply:
wasmer is not there in swc_core crate?